src/App.tsx

```diff
@@ -1,61 +1,107 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
 
 export const App = () => {
+  const [todos, setTodos] = useState([...todosFromServer]);
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('');
+  const [users] = useState([...usersFromServer]);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    let largestId = 0;
+
+    todos.forEach(t => {
+      if (t.id > largestId) {
+        largestId = t.id;
+      }
+    });
+
+    const todo = {
+      id: largestId + 1,
```

Comment:
it would be better to apply id in this way
```suggestion
      id: Math.max(...todoList.map((todo) => todo.id)) + 1,
```